js/base/main.js

```diff
@@ -283,8 +283,15 @@ export class RipeCommonsMainPlugin extends RipeCommonsPlugin {
             // runs the remote business logic to obtain the multiple
             // target groups available for initials as well as the
             // available characters for personalization
-            const [groups, supportedCharacters] = await Promise.all([
+            const [

```

Comment:
@gcandal The `refreshInitialsData` must be done using a lazy-loaded approach meaning that it's only called the first time it's needed for a certain model nad, not as a result of the initial config call on a model

Reply:
This way we should avoid extra HTTP call at boot that can result in 404 or any other kind of error due to no implementation

Reply:
https://github.com/ripe-tech/ripe-commons-pluginus/pull/184